app/_hub/kong-inc/ldap-auth-advanced/index.md

```diff
@@ -31,84 +33,92 @@ params:
+        TCP port where the LDAP server is listening. 389 is the default
+        port for non-SSL LDAP and AD. 686 is the port required for SSL LDAP and AD. If `ldaps` is
+        configured, you must use port 686.
     - name: ldap_password
       required:
       default:
       value_in_examples:
       description: |
-        The password to the LDAP server
+        The password to the LDAP server.
     - name: start_tls
-      required:
+      required: true
       default: "`false`"
-      value_in_examples:
+      value_in_examples: true
       description: |
-        Set it to `true` to issue StartTLS (Transport Layer Security) extended operation over `ldap` connection
+        Set it to `true` to issue StartTLS (Transport Layer Security) extended operation
+        over `ldap` connection. If the `start_tls` setting is enabled, ensure the `ldaps`
+        setting is disabled.
     - name: ldaps
-      required:
+      required: true
       default: "`false`"
       value_in_examples:
       description: |
         Set it to `true` to use `ldaps`, a secure protocol (that can be configured
-        to TLS) to connect to the LDAP server.
+        to TLS) to connect to the LDAP server. When `ldaps` is
+        configured, you must use port 686. If the `ldap` setting is enabled, ensure the
+        `start_tls` setting is disabled.
     - name: base_dn
       required: true
       default:
       value_in_examples: dc=example,dc=com
       description: |
-        Base DN as the starting point for the search; e.g., "dc=example,dc=com"
+        Base DN as the starting point for the search; e.g., "dc=example,dc=com".
     - name: verify_ldap_host
-      required:
+      required: true
       default: "`false`"
-      value_in_examples:
+      value_in_examples: false
       description: |
-        Set it to `true` to authenticate LDAP server. The server certificate will be verified according to the CA certificates specified by the `lua_ssl_trusted_certificate` directive.
+        Set to `true` to authenticate LDAP server. The server certificate will be verified according to the CA certificates specified by the `lua_ssl_trusted_certificate` directive.
     - name: attribute
       required: true
       default:
       value_in_examples: cn
       description: |
-        Attribute to be used to search the user; e.g., "cn"
+        Attribute to be used to search the user; e.g., "cn".
     - name: cache_ttl
-      required:
+      required: true
       default: "`60`"
-      value_in_examples:
+      value_in_examples: 60
       description: |
-        Cache expiry time in seconds
+        Cache expiry time in seconds.
     - name: timeout
       required: false
       default: "`10000`"
       value_in_examples:
       description: |
-        An optional timeout in milliseconds when waiting for connection with LDAP server
+        An optional timeout in milliseconds when waiting for connection with LDAP server.
```

Comment:
```suggestion
        An optional timeout, in milliseconds, when waiting for connection with LDAP server.
```

Reply:
These commas are optional and prefer not to have these speedbumps to the reader's eye.

stet